Tools/Scripts/run-mvt-tests

```diff
@@ -0,0 +1,305 @@
+        except CalledProcessError:
+            return False
+        return True
+
+    def _isFlatpakSDK(self):
+        return not os.getenv("WEBKIT_JHBUILD") or os.getenv("WEBKIT_JHBUILD") == 0
+
+    def _isJHBuild(self):
+        return os.getenv("WEBKIT_JHBUILD") and os.getenv("WEBKIT_JHBUILD") == 1
+
+    def _run(self, cmd, env):
+        def dict_flat(d):
+            return " ".join([f"{key}={value}" for key, value in d.items()])
+        try:
+            env = {
+                "WEBKIT_EXEC_PATH": "/app/webkit/WebKitBuild/{}/bin".format(self.configuration.capitalize()),
+                "WEBKIT_INJECTED_BUNDLE_PATH": "/app/webkit/WebKitBuild/{}/lib".format(self.configuration.capitalize()),
+                "COG_MODULEDIR": "/app/webkit/WebKitBuild/{}/Tools/cog-prefix/src/cog-build/platform".format(self.configuration.capitalize()),
+            }
+            subcommand = "{} {} --host=all --port={}".format(dict_flat(env), self.binary, self.port)
+            cmd.append(subcommand)
+            subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        except CalledProcessError:
+            sys.exit(1)
+        time.sleep(2)
+
+
+def main():
+    """Script entry point."""
+
+    args = parse_args()
+
+    # 1. Start WPEWebDriver.
+    print("Start WPEWebDriver:", end=' ')
+    wpeWebDriver = WPEWebDriver(args)
+    wpeWebDriver.start()
+    print("OK")
+        
+    # 2. Start the WebDriver
+    args.browser = os.path.join("/home/dpino/workspace/gh/webkit", "WebKitBuild/WPE/Release/Tools/cog-prefix/src/cog-build/launcher/cog")
+    print("Connecting to remote driver:", end=' ')
+    driver = webdriver.Remote(
+        command_executor=args.address,
+        options=options(args.browser, args.platform),
+    )
+    print("OK")
+
+    server = os.getenv("MVT_INSTANCE_ADDRESS", "https://mvt.rdkcentral.com")
+    print(f"Using MVT server: {server}")
+
+    # 3. Load the test suite with parameter `command=run`
+    print("Loading MVT:", end=' ')
+    driver.get(f"{server}/?test_type={args.suite}&command=run")
+    print("OK")
+    print("Waiting for globalRunner instantiation:", end=' ')
+    WebDriverWait(driver, 20).until(suite_has_started)
+    print("OK")
+
+    # 4. Wait for test to finish
+    print("Waiting for test run to finish:", end=' ')
+    WebDriverWait(driver, 3600).until(suite_has_finished)
+    print("OK")
+
+    # 5. Fetch with getMvtTestResults()
+    print("Fetching the test results:", end=' ')
+    results = driver.execute_script("return getMvtTestResults()")
+    print("OK")
+
+    # 6. Parse JSON test results
+    print("Saving the test results:", end=' ')
+    with open("results.json", "w", encoding="utf-8") as handle:
+        json.dump(results, handle, indent=True)
+    print("OK")
+
+    # 7. Analyze results.
+    total, failed, passed = analyze_results('results.json')
+    print("Ran {} tests of which {} failed".format(total, failed))
```

Comment:
Here I think it will be helpful also to print the names of the test that failed, so that get registered on the logs.
This will be helpful when checking later the buildbot logs we can see which tests failed on each run.